app/controllers/main_controller.rb

```diff
@@ -189,14 +189,14 @@ def redirectit(release)
 
 
   def developer
-    #redirectit("developer")
-    #return
-    flash.now[:warn] = _("There is no openSUSE release in testing phase at the moment. <br/>" +
-        " If you want to use bleeding edge software, please use <a href='http://en.opensuse.org/Portal:Tumbleweed'>openSUSE Tumbleweed</a>.")
-    @exclude_debug = true
-    @include_home = 'false'
-    set_release("421")
-    render :template => "main/release"
+    redirectit("developer")
+    return
+#    flash.now[:warn] = _("There is no openSUSE release in testing phase at the moment. <br/>" +
+#        " If you want to use bleeding edge software, please use <a href='http://en.opensuse.org/Portal:Tumbleweed'>openSUSE Tumbleweed</a>.")
+#    @exclude_debug = true
+#    @include_home = 'false'
+#    set_release("421")
+#    render :template => "main/release"
```

Comment:
If @ancorgs says so, I will agree then : )

But usually we can do this too (ask @ancorgs' opinion about that)

```ruby
if false
  # NOTE: Enable this for the release
  flash.now[:wan] = ...
  ...
end
redirectit("developer")
return
```

This have the advantage of: be explicit about the intention, and less changes when the release comes.

Anyway, @ancorgs rulez here : )))